web/admin/src/App.css

```diff
@@ -69,7 +69,7 @@ ul {
 
 .containerRow {
   display: flex;
-  flex-direction: nowrap;
+  flex-direction: row wrap;
```

Comment:
Do you mean https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow ?  `flex-direction` does not allow this https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction